goicicle/setup.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
```

Comment:
there is a typo on line 39:

```
    echo "BN254_PATH found @ $TARGET_BLS12_381_PATH"
    export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$TARGET_BLS12_381_PATH
```

BN254_PATH should be BLS12_381_PATH

Reply:
fixed